src/content/LibraryContent.tsx

```diff
@@ -10,6 +10,13 @@ export const libraries: Library[] = [
     repository: 'https://github.com/process-analytics/bpmn-visualization-js',
     type: 'TypeScript',
   },
+  {
+    name: 'BPMN Visualization - R package',
+    description:
+      'A POC to embed the BPMN Visualization library in an R package',
```

Comment:
I wouldn't mention that this is a poc here. Why not stating 'experimental implementation'?